Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rust-algorand"
-version = "1.7.0"
+version = "1.7.1"
```

Comment:
This should be a minor version bump, not a patch version bump.

Reply:
What's the new feature? isn't this PR just a fix?

Reply:
Solved in private.